Gemfile.lock

```diff
@@ -1075,4 +1075,4 @@ RUBY VERSION
    ruby 2.6.6p146
 
 BUNDLED WITH
-   1.17.3
+   1.17.3
```

Comment:
Looks like a typo here

Reply:
Somehow an older version of Gemfile.lock got pushed to this branch so I replaced it with the current one in master.

Don't those changes cancel each other out? (I don't know why it looks like that.)

Reply:
Is there an extra newline or space possibly?

Reply:
No, I checked.

Reply:
I just copied/pasted/pushed it again - https://github.com/department-of-veterans-affairs/vets-api/blob/master/Gemfile.lock